streaming-serverless-fraud-detection/README.md

```diff
@@ -0,0 +1,210 @@
+# Real-time transaction fraud detection/prevention using Serverless and Amazon Fraud Detector
+
+This example demonstrates a serverless approach to detect online transaction fraud in near real-time. It shows how detection can be plugged into various data streaming and event- driven architectures, depending on the outcome to be achieved and actions to be taken to prevent fraud - alert the customer/user about the fraud, flag the transaction for additional review, etc.
+
+This sample implements three architectures:
+ - [Streaming data inspection and fraud detection/prevention](#streaming-data-inspection-and-fraud-detectionprevention) - using Amazon Kinesis Data Stream, AWS Lambda, AWS Step Functions, and Amazon Fraud Detector
+ - [Streaming data enrichment for fraud detection/prevention](#streaming-data-enrichment) - using Amazon Kinesis Data Firehose, AWS Lambda, and Amazon Fraud Detector
+ - [Event data inspection and fraud detection/prevention](#event-data-inspection) - using Amazon EventBridge, AWS Step Functions, and Amazon Fraud Detector
+
+## Streaming data inspection and fraud detection/prevention
```

Comment:
Is there a reason that you use the `detection/prevention` combination throughout? Perhaps this is industry specific terminology? If not can we just simplify to use `detection`?

Reply:
So you can brush up on service name usage policy: 
https://w.amazon.com/bin/view/AWSDocs/editing/service-names

Reply:
Service names updated